wpilibj/src/athena/cpp/lib/NotifierJNI.cpp

```diff
@@ -141,34 +141,34 @@ Java_edu_wpi_first_wpilibj_hal_NotifierJNI_initializeNotifier(
   notify->Start();
   notify->SetFunc(env, func, mid);
   int32_t status = 0;
-  void *notifierPtr = initializeNotifier(notifierHandler, notify, &status);
+  HalNotifierHandle notifierHandle = initializeNotifier(notifierHandler, notify, &status);
 
-  NOTIFIERJNI_LOG(logDEBUG) << "Notifier Ptr = " << notifierPtr;
+  NOTIFIERJNI_LOG(logDEBUG) << "Notifier Handle = " << notifierHandle;
   NOTIFIERJNI_LOG(logDEBUG) << "Status = " << status;
 
-  if (!notifierPtr || !CheckStatus(env, status)) {
+  if (!notifierHandle || !CheckStatus(env, status)) {
```

Comment:
Can't notifierHandle be negative as well as 0 to indicate an error?  (e.g. shouldn't this be `notifierHandle <= 0`)?